### ISSUE TITLE
Any way to make files open in Franz instead of tabs

### ISSUE THREAD

**Issue by asmerkin** [kabeersvohra/googledrivefranz#1]
It would be nice to be able to upload files in Tabs instead of new Chrome pages.

**Comment by kabeersvohra** [comment]
If you right click and then preview a file it should open up in Franz directly. Is this what you were looking for?

**Comment by asmerkin** [comment]
That works partially .. I’d love to be able to edit files ( spreadsheets ) within Franz.

On Wed, Jun 6, 2018 at 6:17 AM, Kabeer Vohra <notifications@github.com> wrote:
If you right click and then preview a file it should open up in Franz directly. Is this what you were looking for?

**Comment by kabeersvohra** [comment]
Yeah sorry, that would require google docs integration within Franz :/

**Comment by asmerkin** [comment]
I’m also reading the code …
Do you know if there’s a way to add an event listener to force Drive to open files within the Electron webview? Maybe in a new window.

On Wed, Jun 6, 2018 at 12:06 PM, Kabeer Vohra <notifications@github.com> wrote:
Yeah sorry, that would require google docs integration within Franz :/
